examples/win32-front-proxy/docker-compose.yaml

```diff
@@ -0,0 +1,38 @@
+version: "3.7"
+services:
+
+  front-envoy:
+    build:
+      context: .
+      dockerfile: Dockerfile-frontenvoy
+    networks:
+      - envoymesh
+    ports:
+      - "8080:8080"
+      - "8443:8443"
+      - "8003:8003"
```

Comment:
nit: since this is here, maybe the example can use this exposed port instead of entering the container

Reply:
We do curl requests from both inside the container and from the host.